app/components/GameButtonsPanel/GameButtonsPanel.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ButtonGroup from '../ButtonGroup/ButtonGroup';
+
+export default function GameButtonsPanel(props) {
+  const {
+    startingPoint, endingPoint, startButtonPressed, endButtonPressed,
+  } = props.state;
+  const checkIfDisabled = (location) => {
```

Comment:
'check' is redunant. `isLocationDisabled` is better, though 'Location' is ambiguous.

Reply:
done